Fixed #22569 -- Made ModelAdmin.lookup_allowed() respect get_list_filter(). (#16759)
django/django: docs/internals/deprecation.txt

* Support for passing positional arguments to ``BaseConstraint`` will be
  removed.

> * Support for ``ModelAdmin.lookup_allowed`` without argument ``request`` will
>   be removed.

Reviewer: ```suggestion
* ``request`` will be required in the signature of
  ``ModelAdmin.lookup_allowed()`` subclasses.
```
